molecule/default/python.yml

```diff
@@ -4,6 +4,9 @@
   become: yes
```

Comment:
Update the play name directly above this line.

Reply:
Please see [1f58866](https://github.com/cisagov/ansible-role-cyhy-core/pull/65/commits/1f588661f2703dfe54c9892e458a232e78bf0a72).